web_hide_custom_search/static/src/js/custom_filter_patch.esm.js

```diff
@@ -0,0 +1,24 @@
+odoo.define("web_hide_custom_search.web.CustomFilterItemPatch", function (require) {
+    const CustomFilterItem = require("web.CustomFilterItem");
+    const {patch} = require("web.utils");
+    const {hooks, useState} = owl;
+    const {onWillStart} = hooks;
+    const rpc = require("web.rpc");
+
+    patch(CustomFilterItem.prototype, "web_hide_custom_search.CustomFilterItemPatch", {
+        setup() {
+            this._super.apply(this, arguments);
+            this.visibleState = useState({isVisible: true});
```

Comment:
This may be more in line with the usual convention?
```suggestion
            this.state = useState({isVisible: true});
```

Reply:
I checked using state, but since it's initialized inside the constructor, even if we try to add visible inside setup(), it only contains the original initialized value. So, I’ll just use a new variable instead.